rasa/model.py

```diff
@@ -105,11 +105,11 @@ def add_evaluation_file_to_model(
     # add model file to folder
     if data_format == "json":
         data_path = os.path.join(tmpdir, "data.json")
-        with open(data_path, "w") as f:
+        with open(data_path, "w", encoding="utf-8") as f:
             f.write(json.dumps(payload))
```

Comment:
In these and other `json.dumps()` calls we don't need `ensure_ascii=False`?

Reply:
Hm that's a very good question, let me test

Reply:
looks like this function isn't even used anywhere, haha. but I fixed the other instance because you were correct